Makefile

```diff
@@ -23,6 +23,7 @@ manifests: install-tools ## Generate manifests e.g. CRD, RBAC etc.
 # this is temporary workaround due to issue https://github.com/kubernetes/kubernetes/issues/91395
 # the hack ensures that "protocal" is a required value where this field is listed as x-kubernetes-list-map-keys
 # without the hack, our crd doesn't install on k8s 1.18 because of the issue above
+	./hack/remove-override-descriptions.sh
```

Comment:
The comment no longer matches the command it's for. Would you mind moving `./hack/patch-crd.sh` above `./hack/remove-override-descriptions.sh`?

Reply:
Sure. Done